packages/driver/src/cy/multi-domain/index.ts

```diff
@@ -29,15 +29,15 @@ export function addCommands (Commands, Cypress: Cypress.Cypress, cy: Cypress.cy,
     Cypress.backend('ready:for:domain')
   }
 
-  communicator.on('delaying:html', () => {
+  communicator.on('delaying:html', (request) => {
     // when a secondary domain is detected by the proxy, it holds it up
     // to provide time for the spec bridge to be set up. normally, the queue
     // will not continue until the page is stable, but this signals it to go
     // ahead because we're anticipating multi-domain
     // @ts-ignore
-    cy.isAnticipatingMultiDomain(true)
+    cy.isAnticipatingMultiDomain(request.href)
```

Comment:
this will be useful for error messages when we want to describe where we ended up in cross domain errors.

Reply:
I wonder if we want to change this to something like `cy.isAnticipatingMultiDomainFor(request.href)` to help identify that `isAnticipatingMultiDomain` is namespaced per domain configuration.

Reply:
this has been updated